tests/basicScenarioTest.m

```diff
@@ -0,0 +1,153 @@
+			testCase.Config.Ue.noiseFigure = 7;
+			testCase.Config.Ue.antennaGain = 0;
+            %Set mobility
+            testCase.Config.Mobility.scenario = 'pedestrian';
+			testCase.Config.Mobility.step = 0.01;
+            testCase.Config.Mobility.seed = 19;
+            %Set terrain
+            testCase.Config.Terrain.buildingsFile = 'mobility/buildings.txt';
+			testCase.Config.Terrain.heightRange = [20,50];
+			testCase.Config.Terrain.buildings = load(testCase.Config.Terrain.buildingsFile);
+			testCase.Config.Terrain.buildings(:,5) = randi([testCase.Config.Terrain.heightRange],[1 length(testCase.Config.Terrain.buildings(:,1))]);
+			testCase.Config.Terrain.area = [...
+				min(testCase.Config.Terrain.buildings(:, 1)), ...
+				min(testCase.Config.Terrain.buildings(:, 2)), ...
+				max(testCase.Config.Terrain.buildings(:, 3)), ...
+				max(testCase.Config.Terrain.buildings(:, 4))];
+            %Set traffic to fullbuffer, to ensure that traffic is transmitted
+            testCase.Config.Traffic.primary = 'fullBuffer';
+            testCase.Config.Traffic.mix = 0;
+            %Set physical layer
+            testCase.Config.Phy.uplinkFrequency = 1747.5;
+			testCase.Config.Phy.downlinkFrequency = 1842.5;
+            %Consider if this enough
+            %set Channel properties
+            testCase.Config.Channel.mode = '3GPP38901';
+			testCase.Config.Channel.fadingActive = true;
+			testCase.Config.Channel.interferenceType = 'Full';
+			testCase.Config.Channel.shadowingActive = true;
+			testCase.Config.Channel.reciprocityActive = true;
+			testCase.Config.Channel.perfectSynchronization = true;
+			testCase.Config.Channel.losMethod = '3GPP38901-probability';
+			testCase.Config.Channel.region = struct('type', 'Urban', 'macroScenario', 'UMa', 'microScenario', 'UMi', 'picoScenario', 'UMi');
+            %Set scheduling
+            testCase.Config.Scheduling.type = 'roundRobin';
+            %Son Parameters
+            testCase.Config.Son.neighbourRadius = 100;
+			testCase.Config.Son.hysteresisTimer = 0.001;
+			testCase.Config.Son.switchTimer = 0.001;
+			testCase.Config.Son.utilisationRange = 1:100;
+			testCase.Config.Son.utilLow = Son.utilisationRange(1);
+			testCase.Config.Son.utilHigh = Son.utilisationRange(end);
+			testCase.Config.Son.powerScale = 1;
+
+            %Set HARQ and ARQ
+            testCase.Config.Harq.active = true;
+			testCase.Config.Harq.maxRetransmissions = 3;
+			testCase.Config.Harq.redundacyVersion = [1, 3, 2];
+			testCase.Config.Harq.processes = 8;
+            testCase.Config.Harq.timeout = 3;
+            testCase.Config.Arq.active = true;
+			testCase.Config.Arq.maxRetransmissions = 1;
+			testCase.Config.Arq.maxBufferSize = 1024;
+            testCase.Config.Arq.timeout = 20;
+            
+            testCase.Simulation = Monster(testCase.Config);
+        end
+    end
+
+    methods (TestMethodTeardown)
+        function resetObjects(testCase)
+            %Test Teardown
+            clear all;
```

Comment:
I'm pretty sure clear all is called regardless of the teardown method.

Reply:
It is now removed